src/components/timeline.js

```diff
@@ -28,12 +30,14 @@ export default class Timeline extends React.Component {
         <ul className='tabs clearfix'>
           <TweetTabContainer account={this.props.account} tab='home'>Timeline</TweetTabContainer>
           <MentionTabContainer account={this.props.account} tab='mentions'>Mentions</MentionTabContainer>
+          <FavoritesTabContainer account={this.props.account} tab='favorites'>Favorites</FavoritesTabContainer>
```

Comment:
>  In my opinion, a frequently updated tab should be placed to left. 

I actually agree to your opinion :+1: The order of tabs in my PR is not really important.
My actual thought on favorite list feature was, like an official app, make it able to see the list from the user profile page. Currently user profile feature is not implemented, so I just put the list as a tab :)

![fav](https://cloud.githubusercontent.com/assets/982850/19153243/a7bd1858-8c10-11e6-9354-07755613336a.png)